### ISSUE TITLE
Swap `-f` to string type from boolean for custom log location

### ISSUE THREAD

**Issue by shadowbq** [ethanpil/sheret#1]
```shell
invalid boolean value "'logs\\sheret.log'" for -f: strconv.ParseBool: parsing "'logs\\sheret.log'": invalid syntax
```

Sheret uses a hard-coded location for log files which is inconvenient for log purposes. It writes the log **_into the current working directory_**, **not next to executable** as described in the `readme.md`. 

I can not write the log to a **sane** location such as `c:\windows\temp` or any other location which might be provided by environmental variables.

https://github.com/ethanpil/sheret/blob/347c0ba0cf7a1c43d284f476c806513d3406abaf/src/sheret.go#L65

**Comment by ethanpil** [comment]
Good point. Thanks for the feedback. Can you submit a PR? Otherwise I will take a look at this and fix over the next few weeks.

**Comment by shadowbq** [comment]
If I have time Ill send a pull request

**Comment by ethanpil** [comment]
@shadowbq any interest still in the PR?

**Comment by ethanpil** [comment]
Thanks for your patience. This has been addressed with the 1.1 release here: https://github.com/ethanpil/sheret/releases/tag/1.1
